content/en/docs/reference/glossary/cidr.md

```diff
@@ -0,0 +1,18 @@
+---
+title: CIDR
+id: cidr
+date: 2019-11-12
+full_link: 
+short_description: >
+  CIDR is a notation for describing blocks of IP addresses and is used heavily in various networking configurations.
+
+aka:
+tags:
+- networking
+---
+CIDR (Classless Inter-Domain Routing) is a notation for describing blocks of IP addresses and is used heavily in various networking configurations.
+
+<!--more-->
+
+In the context of Kubernetes, each {{< glossary_tooltip text="Node" term_id="node" >}} is assigned a range of IP addresses through the start address and a subnet mask using CIDR. This allows Nodes to assign each {{< glossary_tooltip text="Pod" term_id="pod" >}} a unique IP address. Although originally a concept for IPv4, CIDR has also been expanded to include IPv6. 
```

Comment:
Kubernetes uses CIDR blocks for various things including this example.
`ipBlock` from NetworkPolicy also refers to blocks of IP addresses.

Is it helpful to show an actual example, eg mention 203.0.113.128/25 and show the range that it covers?

Reply:
Ok, maybe something then to the effect of: "CIDR is used in various networking aspects in Kubernetes. For example..."

I'm not sure providing an example would be useful, in my experience CIDR isn't super intuitive to people who have never seen it before so the example would probably be lost on them. I think there would be two main audiences for this glossary: people who don't know what CIDR is or people who know what CIDR is, but want to know how it applies to Kubernetes. Right now I think the definition helps more of the second audience and the discussion is how much should we cater towards the first audience (complete beginners to CIDR)

Reply:
How does SIG Docs feel about hyperlinking to, eg, Wikipedia's page about this?

Reply:
My feeling is that we don't need to dive into the details of CIDR which is pretty common an acronym in the community.